README.md

```diff
@@ -5,9 +5,19 @@
 [<img src="https://img.shields.io/maven-central/v/org.scala-lang.modules/scala-swing_2.12.svg?label=latest%20release%20for%202.12"/>](http://search.maven.org/#search%7Cga%7C1%7Cg%3Aorg.scala-lang.modules%20a%3Ascala-swing_2.12)
 [![Stories in Ready](https://badge.waffle.io/scala/scala-swing.svg?label=ready&title=Ready)](http://waffle.io/scala/scala-swing)
```

Comment:
Let's remove this link, as Waffle is closing its doors: https://waffle.io/closing-its-doors

Reply:
done in d4edd9ba0427ad6d621f3c123032948c8ba0b0fa